proxy/src/coco.rs

```diff
@@ -1,7 +1,8 @@
 //! Abstractions and utilities for git interactions through the API.
 
-pub use librad::uri::rad_urn::ParseError;
-pub use librad::uri::RadUrn as Urn;
+pub use librad::hash::Hash;
+pub use librad::meta::project::Project;
+pub use librad::uri::{rad_urn::ParseError, Path, Protocol, RadUrn as Urn};
```

Comment:
I wouldn't expose the `ParseError` at the top level. There are multiple error types that are called `ParseError` in librad. I would probably say the same for `Path` and `Protocol`, as well.

Why did you decide to export them here?

Note that this was ok for `RadUrn` because it's unique in the namespace.

Reply:
Ah ok. I think I was unsure about why we import from `librad` in some instances but export `librad` entities from `coco` elsewhere. The `ParseError` was already exported here, should it be removed?

Reply:
We can do a safe middle ground and export `self` so they can be accessed via `coco::uri`